var/spack/repos/builtin/packages/python/package.py

```diff
@@ -752,11 +755,61 @@ def config_vars(self):
 """ % self.print_string("json.dumps(config)")
 
         dag_hash = self.spec.dag_hash()
+
         if dag_hash not in self._config_vars:
+            # Default config vars
+            version = self.version.up_to(2)
+            try:
+                cc = self.compiler.cc
+                cxx = self.compiler.cxx
+            except (TypeError, NoCompilerForSpecError):
+                cc = 'cc'
+                cxx = 'c++'
+
+            config = {
+                'CC': cc,
+                'CXX': cxx,
+                'INCLUDEPY': self.prefix.include.join('python{}').format(version),
+                'LIBDEST': self.prefix.lib.join('python{}').format(version),
+                'LIBDIR': self.prefix.lib,
+                'LIBPL': self.prefix.lib.join('python{0}').join(
+                    'config-{0}-{1}').format(version, sys.platform),
+                'LDLIBRARY': 'libpython{}.{}'.format(version, dso_suffix),
+                'LIBRARY': 'libpython{}.a'.format(version),
+                'LDSHARED': cc,
+                'LDCXXSHARED': cxx,
+                'PYTHONFRAMEWORKPREFIX': '/System/Library/Frameworks',
+                'prefix': self.prefix,
+                'config_h_filename': self.prefix.include.join('python{}').join(
+                    'pyconfig.h').format(version),
+                'makefile_filename': self.prefix.lib.join('python{0}').join(
+                    'config-{0}-{1}').Makefile.format(version, sys.platform),
+                'python_inc': {
+                    # plat_specific
+                    'true': os.path.join('include64', 'python{}'.format(version)),
```

Comment:
Never seen an `include64` dir before, but `get_python_inc` has a `plat_specific` flag for a reason, so why not check both.